zulipterminal/model.py

```diff
@@ -139,9 +140,10 @@ def __init__(self, controller: Any) -> None:
         unicode_emoji_data = unicode_emojis.EMOJI_DATA
         for name, data in unicode_emoji_data.items():
             data['type'] = 'unicode_emoji'
+        typed_unicode_emoji_data = cast(NamedEmojiData, unicode_emoji_data)
```

Comment:
This can't be done inline right?

Reply:
You've modified the original, so this seems reasonable :+1: